src/beneuro_data/conversion/anipose_interface.py

```diff
@@ -73,25 +84,30 @@ def add_to_nwbfile(
         nwbfile: NWBFile,
         metadata: Optional[DeepDict] = None,
         stub_test: bool = False,
+        use_default_fps: bool = True,
     ):
-        try:
-            timestamps = self.get_original_timestamps()
-            starting_time = None
-            rate = None
-        except:
-            # if we can't get the timestamps from spikeglx for whatever reason
-            # then just assume that the frames come with DEFAULT_FPS
-            # and that the first one is at t=0 which should be true
+
+        # Allignment: As cameras automatically start recording, I am setting
+        # the default option to be starting_time=0.0 and DEFAULT_FPS
```

Comment:
```suggestion
        # Alignment: As cameras start recording when PyControl sends them the signal at t=0,
        # and in theory sends a signal with DEFAULT_FPS frequency, set the default option for the
        # timing of the frames to use `starting_time` and `rate` instead of explicit timestamps.
```